spec/OBD/OBD.vspec

```diff
@@ -465,31 +464,31 @@
   description: PID 3B - Lambda current for wide range/band oxygen sensor 8
 
 - CommandedEGR:
-  datatype: uint8
+  datatype: float
   type: sensor
   unit: percent
   description: PID 2C - Commanded exhaust gas recirculation (EGR)
 
 - EGRError:
-  datatype: uint8
+  datatype: float
   type: sensor
   unit: percent
   description: PID 2D - Exhaust gas recirculation (EGR) error
 
 - CommandedEVAP:
-  datatype: uint8
+  datatype: float
   type: sensor
   unit: percent
   description: PID 2E - Commanded evaporative purge (EVAP) valve
 
 - FuelLevel:
-  datatype: uint8
+  datatype: float
   type: sensor
   unit: percent
   description: PID 2F - Fuel level in the fuel tank
 
 - WarmupsSinceDTCClear:
-  datatype: uint16
+  datatype: uint8
```

Comment:
According to https://en.wikipedia.org/wiki/OBD-II_PIDs PID 30 use 1 byte and supports values 0-255 so uint8 shall be sufficient.